src/test/scala/io/dockstore/Requests.scala

```diff
@@ -129,40 +146,40 @@ object Requests {
   object Workflow {
     def getPublishedWorkflow(repo: String) = {
       http("Published Worfklow")
-        .get(s"/workflows/path/workflow/${repo}/published")
+        .get(s"/api/workflows/path/workflow/${repo}/published?services=false")
```

Comment:
This endpoint now has an additional [`subclass` parameter](https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L1055) and the `services` parameter is hidden and deprecated. Wonder if we should move away from using `services`?

Reply:
66703488e93f6dd5338b22baf00f361e639fd4ac